nntrainer/utils/util_func.cpp

```diff
@@ -197,4 +197,12 @@ bool istrequal(const std::string &a, const std::string &b) {
   });
 }
 
+char *get_realpath (const char *__restrict __name, char *__restrict __resolved) {
+#ifdef _WIN32
```

Comment:
How about change the def for the consistency.
for Android, nntrainer uses ```__ANDROID___```, Tizen, ```___TIZEN__```.
So.. for Windows, how about ```___WIN___``` ?

Reply:
Thank you, I didn't know that there was a predefined keyword called "__WIN32__". I will fix it!

Reply:
_WIN32 is predefined macro defined by toolchain. __ANDROID__ and __TIZEN__ are macro defined by nntrainer. If you want __WIN32__, you will need to do it in a common header: ```#define __WIN32__ (_WIN32)``` (do we really need this?)

Reply:
This modification has not been reflected yet. If it is necessary to modify the keyword `_WIN32` into `__WIN32__`, then I will modify and reflect it again.

Reply:
OR... maybe https://en.cppreference.com/w/cpp/filesystem/absolute is better?

Reply:
> _WIN32 is predefined macro defined by toolchain 
if it is defined by Visual Studio, then it is ok to go.

Reply:
> OR... maybe https://en.cppreference.com/w/cpp/filesystem/absolute is better?

The "absolute" function seems to work well in windows. It seems to be best way to use the absolute function. I will reflect this modification.

Reply:
Modifications were reflected to use the "absolute" function. At this time, the existing PR has been modified as it is, but I wonder if it would be better to upload new PR in the future if it is completely different from the existing PR.

Reply:
Filesystem including absolute function does not seem to be fully supported in android r21 (supported from r22) : https://github.com/android/ndk/issues/609#issuecomment-631918497. Therefore, I would like to modify it again by defining the "getRealpath" function as before.

Reply:
I uploaded it reflecting the work situation so far.